docs/src/changelog.md

```diff
@@ -7,6 +7,33 @@ CurrentModule = MathOptInterface
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.19.0 (August 7, 2023)
+
+### Added
+
+ - Added [`Utilities.eval_variables`](@ref) support for
+   [`ScalarNonlinearFunction`](@ref) (#2218) (#2219)
+ - Added support for [`ScalarNonlinearFunction`](@ref) in `FileFormats.NL`
+   (#2228) (#2231)
+ - Added [`Bridges.Constraint.ScalarQuadraticToScalarNonlinearBridge`](@ref) (#2233)
```

Comment:
We should revert this and then tag. Then add it back with the function-convert bridges in v1.20